CONTRIBUTING.md

```diff
@@ -79,6 +77,12 @@ In order to run the same unit tests as the CI workflows, you can run unittests l
 make test
 ```
 
+This will run the full suite of core API unittests. However, if you're trying to run some specific unittests, you can do as follows:
```

Comment:
Maybe we should specify that these tests only concern the api and not the client. The current wording suggests that the command will test everything, whereas there is a specific command for the client. Maybe specify the client test command (while specifying that the current test coverage is not complete yet) ?

Reply:
I thought "core API" was specific enough :sweat_smile: